doc/source/modules/gui/plot/plotsignal.rst

```diff
@@ -91,6 +91,7 @@ Both share the following keys:
 
 - 'event': 'curveClicked' or 'imageClicked'
 - 'button': the mouse button that was pressed in 'left', 'middle', 'right'
+- 'item': The plot item object that was clicked
```

Comment:
Pass item in plot curev|image clicked event so event handling does not have to rely on legends.